Chap_API_Reserved_Keys.tex

```diff
@@ -461,41 +517,24 @@ \subsection{Node realm keys}
 Total available physical memory on a node.
 }
 
-\vspace{\baselineskip}
-
-The following attributes only return information regarding the \emph{caller's} node - any node-related qualifiers shall be ignored. In addition, these attributes require specification of the namespace in the target process identifier except where noted - the value of the rank is ignored in all cases.
```

Comment:
Errr...where did all these go? The comment that was removed is actually outdated anyway. You can retrieve the `PMIX_LOCAL_PEERS`, `PMIX_LOCAL_SIZE`, `PMIX_LOCAL_PROCS`, `PMIX_LOCAL_CPUSETS`, and `PMIX_LOCALLDR` for any node. PRRTE routinely provides them.